internal/template/function.go

```diff
@@ -82,3 +84,50 @@ func resourceRequests(pods corev1.PodList, weights string) (float64, error) {
 	}
 	return totalResources, nil
 }
+
+func cpuUtilization(labelArgs ...string) (string, error) {
+
+	cpuUtilizationQueryTemplate := `
+scalar(
+  sum(
+    max(container_cpu_usage_seconds_total{container="", image=""}) by (pod)
```

Comment:
I think this should be increase over the time range not max because pods that aren't patched won't start from container_cpu_usage_seconds_total=0